### ISSUE TITLE
Duplicate registration for EP: com.intellij.remoteServer.type

### ISSUE THREAD

**Issue by MaXal** [viatra2/IDEA-AWS-Elastic-Beanstalk-Integration#20]
Some of our clients are affected by the following error:

```
java.lang.RuntimeException: Duplicate registration for EP: com.intellij.remoteServer.type
 at com.intellij.openapi.extensions.impl.ExtensionsAreaImpl.registerExtensionPoint(ExtensionsAreaImpl.java:287)
 at com.intellij.openapi.extensions.impl.ExtensionsAreaImpl.registerExtensionPoint(ExtensionsAreaImpl.java:131)
 at com.intellij.ide.plugins.IdeaPluginDescriptorImpl.registerExtensionPoints(IdeaPluginDescriptorImpl.java:286)
 at com.intellij.ide.plugins.PluginManagerCore.a(PluginManagerCore.java:1203)
```

See http://youtrack.jetbrains.com/issue/WI-24614

**Comment by viatra2** [comment]
This issue affects "AWS EB Integration for Web Languages" plugin version 0.6.5 for EAPs since version 138.1901 (only latest PhpStorm EAP at the moment).

The extension points "com.intellij.remoteServer.*" were not registered in the web languages JetBrains IDEs, so in order to use them by the plugin we have copied the declarations into our plugin.xml.

Since http://youtrack.jetbrains.com/issue/IDEA-128927 is fixed, respected extension points are registered both in the IDE and the plugin which brings the duplicate registration exception.
There is no known workaround for user except the plugin deletion. 
The issue will be fixed with the next plugin update as soon as possible.

**Comment by viatra2** [comment]
Fixed in 0.7.1
